subresource-loading.bs

```diff
@@ -167,21 +184,27 @@ To <dfn id="concept-fetch-from-web-bundle">fetch from web bundle</dfn> given
 
 2. Return null.
 
+Note: Returning null here can fallback to subsequent web bundles in
+[=environment settings object/web bundle list=], HTTP cache and ordinal network
```

Comment:
Remove "subsequent web bundles in web bundle list,"? (Since step 1 iterates over the web bundle list)

Reply:
Thanks, done.